sunpy/map/mapbase.py

```diff
@@ -198,7 +198,8 @@ def __init_subclass__(cls, **kwargs):
         cls.__doc__ += textwrap.indent(_notes_doc, "    ")
 
         if hasattr(cls, 'is_datasource_for'):
-            cls._registry[cls] = cls.is_datasource_for
+            if str(cls) != "<class 'pfsspy.map.GongSynopticMap'>":
```

Comment:
Do we want to add a TODO here so we never forget?

Reply:
An issue would be better I think. More visible.